lib/preview/previewUtils.js

```diff
@@ -135,38 +131,30 @@ const trackPreviewEvent = async (action) => {
     accountId
   ).catch(
     (err) => {
-      console.error(`trackUsage failed: ${JSON.stringify(err, null, 2)}`);
+      logger.error(`trackUsage failed: ${JSON.stringify(err, null, 2)}`);
```

Comment:
we should change this to only appear when DEBUG mode is set for the logger (running the cli command in debug mode to surface this). Otherwise, the standard dev won't want to see about this failing in the terminal